package/src/types/types.ts

```diff
@@ -3,9 +3,8 @@ import type { ExtendableGenerics, LiteralStringForUnion } from 'stream-chat';
 export type Asset = {
   duration: number | null;
   filename: string;
-  fileSize: string;
+  fileSize: number | undefined;
   height: number;
-  playableDuration: number | null;
```

Comment:
no need of this, we can map it to duration in native and have it common between expo and native